addons/knowledge/views/knowledge_menus.xml

```diff
@@ -31,13 +31,13 @@
         name="Members"
         action="knowledge.knowledge_article_member_action"
         parent="knowledge_menu_configuration"
-        groups="base.group_no_one"
+        groups="base.group_system"
```

Comment:
So this is a "trick" to get the configuration menu to show up only for admin AND only in debug mode, since you leave group_no_one on the configuration menuitem?

Seems a bit hackish ^^